core/src/main/resources/hudson/model/BooleanParameterDefinition/help-boolean-default_de.html

```diff
@@ -1,3 +1 @@
-<div>
-    Gibt den Vorgabewert für dieses Feld an.
-</div>
\ No newline at end of file
+<div>Gibt den Vorgabewert für dieses Feld an.</div>
```

Comment:
It's kind of inconsistent. Longer text is expanded into multiple lines (where <div> is in separate ones then content), but this one is kept within single one.